core/api/models.py

```
@@ -1,3 +1,81 @@
+from auth_app.models import CustomUser
+
+
+
+class contribution_videos(models.Model):
+    """
+    Model for storing contribution videos.
+    """
+    id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
+    title = models.CharField(max_length=255,null=True,blank=True)
+    video_file = models.FileField(upload_to='contribution_videos/',null=True,blank=True)
+    created_at = models.DateTimeField(auto_now_add=True,)
```

Ruff (0.8.2)

Trailing comma prohibited

Remove trailing comma

(COM819)